Quorum/apis/llms/claude_conversation.py

```diff
@@ -0,0 +1,22 @@
+from Quorum.apis.llms.llm_conversation import LLMConversation
+
+from anthropic import Anthropic
+
+
+class ClaudeConversation(LLMConversation):
+    def __init__(self, api_key: str, model: str = 'claude-3-5-haiku-20241022'):
+        '''
+        Summary
+        -------
+        Initialize a conversation manager for Claude.
+        
+        Parameters
+        ----------
+        api_key : str
+            Your Anthropic API key
+        model : str, default = 'claude-3-5-haiku-20241022'
+            The Claude model to use
+        '''
+        super().__init__()
```

Comment:
Unless you're passing the parameters to the parent, this is useless.

Reply:
There's a also the `messages` field that I want to be initialized. It won't be unless I call the the init on the `LLMCoversation` object.